scripts/deploy/nlp/deploy_triton.py

```diff
@@ -73,18 +74,20 @@ def get_args(argv):
     parser.add_argument("-mil", "--max_input_len", default=256, type=int, help="Max input length of the model")
     parser.add_argument("-mol", "--max_output_len", default=256, type=int, help="Max output length of the model")
     parser.add_argument("-mbs", "--max_batch_size", default=8, type=int, help="Max batch size of the model")
+    parser.add_argument("-mnt", "--max_num_tokens", default=None, type=int, help="Max number of tokens")
+    parser.add_argument("-ont", "--opt_num_tokens", default=None, type=int, help="Optimum number of tokens")
     parser.add_argument(
         "-mpet", "--max_prompt_embedding_table_size", default=None, type=int, help="Max prompt embedding table size"
     )
     parser.add_argument(
         "-upkc", "--use_paged_kv_cache", default=False, action='store_true', help="Enable paged kv cache."
     )
     parser.add_argument(
-        "-dcf",
-        "--disable_context_fmha",
+        "-drip",
+        "--disable_remove_input_padding",
```

Comment:
Could we rather use `--remove_input_padding` instead of `--disable_remove_input_padding` with negations?

Reply:
that would require providing `--remove_input_padding` in customer code to keep the current behavior unchanged (i.e. to use `remove_input_padding=True` when no parameter are specified).

It is only worthwhile if it leads to better naming consistency with TensorRT-LLM code.